src/SALib/util/__init__.py

```diff
@@ -205,6 +206,15 @@ def _nonuniform_scale_samples(params, bounds, dists):
             else:
                 conv_params[:, i] = params[:, i] * (b2 - b1) + b1
 
+        elif dists[i] == "logunif":
+            if b1 >= b2:
```

Comment:
Not sure this is needed. The SciPy underlying function function would raise. But ok if we do want a specific messaging.

Thought for later: we should let errors bubble up and catch all if we want to have a personalized message instead.